### DISCUSSION TITLE
Can power and energy units in results be reported in more readable units (e.g. in MW, KW or GJ)?

### DISCUSSION THREAD

**Discussion by RonaldBerk** [Project-OMOTES/user_feedback#60] [Q&A]
It would be preferable to read the heat production of let's say a Geothermal well in GJ or TJ instead of Wh, since the latter has a lot of zero's and mulitplying them by 1 milion or 1 billion and afterwards dividing them by 3.6 is error sensitive.

**Comment by WouterSpaak** [comment]
This is fairly easy to implement in the frontend. However I would *strongly* suggest that the backend continues using SI units without multipliers, in my experience this is very error prone where one developer/team expects joules and another system returns terrajoules for instance.

**Reply by ewoudwerkman** [reply]
To be clear, everything in ESDL is in SI units. For class attributes, the unit is defined in the schema (e.g. the power attribute of a producer is defined in Watt). For profiles and KPIs this is not fixed, but is defined by the user, exactly because this question: electricity figures are usually in Wh, while heat is in J. 

When defining these units for profiles and KPIs (done in the optimizer and simulator backbends) you can define this including the multiplier (e.g. MW, GJ, kW).  So this is the quick fix.

Our pyESDL python module supports on the fly conversion between units, but this is not integrated in the UI yet, as for the MapEditor we are reworking the visualisation of data. This is planned to be released in the upcoming months.

**Comment by jarsarasty** [comment]
Hi @RonaldBerk ,

A [feature request](https://github.com/Project-OMOTES/user_feedback/issues/70) has been created.
